pkg/apis/mayadata.io/v1alpha1/register.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2020 The MayaData Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	// SchemeBuilder collects the scheme builder functions for the Velero API
+	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
+
+	// AddToScheme applies the SchemeBuilder functions to a specified scheme
+	AddToScheme = SchemeBuilder.AddToScheme
+)
+
+// GroupName is the group name for the Velero API
+const GroupName = "mayadata.io"
```

Comment:
is `dmaas.mayadata.io` better?

Reply:
keeping `mayadata.io` only, since crd is having dmaas prefix.